modules/op-resolver-http.adoc

```diff
@@ -0,0 +1,100 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="resolver-http-specify_{context}"]
+= Specifying a remote pipeline or task with the HTTP Resolver
+
+When creating a pipeline run or task run, you can specify a remote pipeline or task from an HTTP or HTTPS URL.
+
+.Prerequisites
+
+* You can optionally configure the service account for the HTTP resolver.
+
+.Procedure
+
+* To specify a remote pipeline or task from an HTTP URL, use the following reference format in the `pipelineRef` or `taskRef` spec:
++
+[source,yaml]
+----
+# ...
+  resolver: http
+  params:
+  - name: url
+    value: <fully_qualified_http_url>
+  - name: serviceAccount
+    value: <service_account_name> # Optional
+# ...
+----
++
+.Supported parameters for the HTTP Resolver
+|===
+| Parameter | Description | Example Value 
+
+| `serviceAccount`
+| The name of the service account to use when constructing HTTP requests. 
+| `default` 
+
+| `url` 
+| The HTTP URL pointing to the Tekton resource to fetch. 
+| `https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.6/git-clone.yaml` 
```

Comment:
1. I guess that this is supposed to be and example of accepted value. This way it might be misleading.
2. In the past, we were not allowed to reference random GitHub files.

Reply:
@vdemeester can we link upstream catalog task here? Do we have any better sample from an approved repository? Tutorial comes to my mind

Reply:
We could link our own `git-clone` task from the redhat ecosystem (https://github.com/openshift-pipelines/tektoncd-catalog/tree/p/tasks/task-git-clone)

Reply:
Yes, this is much better

Reply:
@buildbricks using https://github.com/openshift-pipelines/tektoncd-catalog/tree/p/tasks/task-git-clone instead works for you ?